### PR TITLE
Improvement/csi 1210 add node static code analysis

### PR DESCRIPTION
**Pull request by oriyarde**
https://golang.org/cmd/vet/

### CODE REVIEW

File: scripts/ci/jenkins_pipeline_csi [hunk 1]
```diff
@@ -14,7 +14,7 @@ pipeline {
                 sh './scripts/run_unitests.sh `pwd`/build/reports'
             }
         }
-        stage ('CSI-node: go Unit testing') {
+        stage ('CSI-node: go local tests') {
```
[the remark this answers is not in the document]
Comment (oriyarde) [reply scripts/ci/jenkins_pipeline_csi]: static code analysis is not a "unit" test

[the remark this answers is not in the document]
Comment (igorgonibm) [reply scripts/ci/jenkins_pipeline_csi]: ok

File: Makefile [hunk 2]
```diff
@@ -28,6 +28,17 @@ GO_TEST_FLAGS=$(shell if [ "$$(uname -m)" = "s390x" ]; then echo "-v"; else echo
 
 .EXPORT_ALL_VARIABLES:
 
+define gofmt-test =
```
Comment (igorgonibm) [inline Makefile]: Handle indentation - For all method body lines add leading tab

Comment (oriyarde) [reply Makefile]: image

**Pull request merged**